src/index.html

```diff
@@ -11,12 +11,58 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com"
+    />
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    />
     <link
+      href="https://fonts.googleapis.com/css2?family=Jost:wght@400;500&family=Roboto:wght@500&display=swap"
       rel="stylesheet"
-      href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="header_logo" href="/">
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. 'header_logo' should be renamed to something more meaningful, such as 'header__logo' if it represents a logo inside the header.